src/components/ChooseIssueOrRevoke/ChooseIssueOrRevoke.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render, screen } from "@testing-library/react";
+import { ChooseIssueOrRevoke } from "./ChooseIssueOrRevoke";
+import { BrowserRouter, useLocation } from "react-router-dom";
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useLocation: jest.fn(),
+}));
+
+const mockUseLocation = useLocation as jest.Mock;
+
+const selectedClassName = "text-blue underline";
+
+describe("ChooseIssueOrRevoke", () => {
+  it("should render component correctly and select 'Revoke Document' button", () => {
```

Comment:
this test is brittle (only testing for css class names) i would rather not include imo.

if want, should use export of `selectedStyle` in component, rather than redefining here as `selectedClassName` in the test (now u have 2 places to update if styles changed)

Reply:
ok agree with u, i removed the test and added in storybook and move this component to UI as it only for the UI and does no other logic.